Artifacts/windows-chocolatey/startChocolatey.ps1

```diff
@@ -0,0 +1,21 @@
+﻿Param(
+    # comma or semicolon separated list of chocolatey packages.
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory=$True)]
+    [string] $RawPackagesList,
```

Comment:
again, minor but inconsistent casing in your local variables. A few cases like this are PascalCased.